tests/NHSD.BuyingCatalogue.Ordering.Api.IntegrationTests/Features/Order Summary 5321.feature

```diff
@@ -52,17 +52,15 @@ Scenario: 3. If the order ID does not exist, return not found
     When the user makes a request to retrieve the order summary with the ID INVALID
     Then a response with status code 404 is returned
 
-@ignore
 @5321
 Scenario: 4. If a user is not authorised then they cannot access the order summary
-    #This will not have the logging in
+    Given no user is logged in
     When the user makes a request to retrieve the order summary with the ID C000014-01
     Then a response with status code 401 is returned
 
-@ignore
 @5321
-Scenario: 5. A non authority user cannot access the order summary
-    #This will have a non authority user logged in
+Scenario: 5. A non buyer user cannot access the order summary
+    And the user is logged in with the Authority role for organisation 4af62b99-638c-4247-875e-965239cd0c48

```

Comment:
Same here. Given rather than an And? Pretty sure it doesn't really matter, but for readability I think it helps.